platform/src/main/java/org/hillview/FactoryBenchmark.java

```diff
@@ -0,0 +1,70 @@
+package org.hillview;
+
+
+import org.hillview.dataset.LocalDataSet;
+import org.hillview.dataset.api.IDataSet;
+import org.hillview.dataset.api.ISketch;
+import org.hillview.sketches.BucketsDescriptionEqSize;
+import org.hillview.sketches.Histogram;
+import org.hillview.sketches.HistogramSketch;
+import org.hillview.table.Table;
+import org.hillview.table.api.ColumnAndConverterDescription;
+import org.hillview.table.api.IColumn;
+import org.hillview.table.api.ITable;
+import org.hillview.table.columns.DoubleArrayColumn;
+import org.hillview.table.membership.DenseMembershipSet;
+import org.hillview.table.membership.FullMembershipSet;
+import org.hillview.table.membership.SparseMembershipSet;
+import org.hillview.utils.HillviewLogger;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+
+public class FactoryBenchmark {
```

Comment:
Please make this a test. The other benchmark files need to have a main for guaranteeing precision.
This one probably does not.

Reply:
But if it is a test you may want to keep it disabled by default, because it takes a really long time.